python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -1007,7 +1104,11 @@ class HourlyPartitionsDefinition(TimeWindowPartitionsDefinition):
             Default is None. Can provide in either a datetime or string format.
         minute_offset (int): Number of minutes past the hour to "split" the partition. Defaults
             to 0.
-        fmt (Optional[str]): The date format to use. Defaults to `%Y-%m-%d`.
+        fmt (Optional[str]): The date format to use. Defaults to `%Y-%m-%d`. Note that if a non-UTC
+            timezone is used, the date format must include a timezone offset to disambiguate between
+            multiple instances of the same time before and after the Fall DST transition. If the
```

Comment:
this is only if there are multiple instances of the same time in the partition set right (i.e. it does not affect daily partition sets), should we call that out here